djstripe/models.py

```diff
@@ -593,6 +563,39 @@ def record_charge(self, charge_id):
         data = stripe.Charge.retrieve(charge_id)
         return Charge.sync_from_stripe_data(data)
 
+    @staticmethod
+    @webhook.handler_all
+    def event_attach_customer(event, data, cat, sub_cat):
+        stripe_customer_crud_events = ["created", "updated", "deleted", ]
+        skip_events = ["plan", "transfer", ]
```

Comment:
I will be happy to adhere to any style convention you guys want. However, I've always intentionally included commas at the end of every dict, tuple, and list as a matter of style. The benefits of this are obvious and well known for multi-line data structures. The benefits in single line lists (such as the one above).... I'm not 100% sure but I still do it there too.

If you have a strong style preference let me know and I will remove, otherwise I'll assume they are harmless